tests/acceptance/CoversCampaignCept.php

```diff
@@ -13,9 +13,9 @@
 	'post_content' => $I->generateGutenberg('wp:planet4-blocks/covers', [
 		'cover_type'  => '2',
 		'title'       => 'Campaign',
-		'tags'        => '6,20',
+		'tags'        => [6, 20],
```

Comment:
Interesting that this test was passing before. It seems like it would try to use a single tag with a comma in it? :thinking:

Reply:
Yeah it's strange... Also the '0' value for `covers_view` doesn't exist, so I guess there was a default case that was applied instead?